### DISCUSSION TITLE
Detect the contents of a search field?

### DISCUSSION THREAD

**Discussion by paul1149** [furybee/chrome-tab-modifier#342] [Q&A] [answered]
Is it possible to detect the contents of a search field? For instance, I'm using startpage.com for a search, but because I use the POST method for privacy, the URL fragment method does not work. {title} just produces "Startpage Search Results".

The search field at Startpage is at ` div#header-search-form div form#search input#q ` on the page. But that is not working.

**Comment by sebastienfontaine** [comment]
Hey!

That's strange because I added a test for the feature with the parameters you provided, and I do get the value of the input returned. Is your site a SPA (Single Page Application), AJAX requests or does it use iframes? This could cause issues since the DOM is loaded dynamically.

![CleanShot 2024-07-06 at 14 30 20@2x](https://github.com/furybee/chrome-tab-modifier/assets/45472257/db1f0af0-4258-41c9-85e8-b29e31095934)

**Reply by paul1149** [reply]
Can't answer except it's not a SPA, it's just a page in my browser.

I searched the Help file and the Readme.md and could not find anything on DOM, though I know I had seen it before somewhere. That string is the only thing in my Title field in the rule. Perhaps I need a DOM delimiter to tell TM what it should look for?

**Reply by paul1149** [reply, marked as answer]
Wow, I simply put that string in curly brackets, and it works perfectly:

{div#header-search-form div form#search input#q}
